ui/app/adapters/pki/issuer.js

```diff
@@ -39,7 +39,8 @@ export default class PkiIssuerAdapter extends ApplicationAdapter {
       const issuerRecord = await this.queryRecord(store, type, { id, backend: query.backend });
       const { data } = issuerRecord;
       const isRoot = await verifyCertificates(data.certificate, data.certificate);
-      return { ...keyInfo, ...data, isRoot };
+      const parsedCertificate = await parseCertificate(data.certificate);
+      return { ...keyInfo, ...data, isRoot, parsedCertificate };
```

Comment:
parsedCertificate data? or can we just grab the `common_name` here?

Reply:
we could just grab the `common_name`. there was extra nesting to do here so i opted to keep all the parsedData info. Since we don't need anything else I agree that we should just get the common name.

Reply:
Nice!